File: src/main/scala/com/besuikerd/autologistics/common/lib/network/ServerMessageHandler.scala
package com.besuikerd.autologistics.common.lib.network

import cpw.mods.fml.common.network.simpleimpl.{MessageContext, IMessage}
import net.minecraft.entity.player.EntityPlayer


trait ServerMessageHandler[REQ <: IMessage, REPLY <: IMessage] extends SidedMessageHandler[REQ, REPLY]{
  override def onClientMessage(player: EntityPlayer, message: REQ, ctx: MessageContext): REPLY = null.asInstanceOf[REPLY]
}

File: src/main/java/com/besuikerd/autologistics/common/lib/network/ServerMessageHandler.java
package com.besuikerd.autologistics.common.lib.network;

import cpw.mods.fml.common.network.simpleimpl.IMessage;
import cpw.mods.fml.common.network.simpleimpl.MessageContext;
import net.minecraft.entity.player.EntityPlayer;

public abstract class ServerMessageHandler<REQ extends IMessage, REPLY extends IMessage> extends SidedMessageHandler<REQ, REPLY> {
    @Override
    public REPLY onClientMessage(EntityPlayer player, REQ message, MessageContext ctx) {
        return null;
    }
}
